includes/src/Report/Table.js

```diff
@@ -127,7 +127,7 @@ class Table extends React.PureComponent {
                             if (output_path !== undefined) {
                                 return (
                                     <td>
-                                        <a href={"/output_files/" + output_path} target="_blank" className="btn btn-primary">Browse</a>
+                                        <a href={"/output_files/" + output_path} target="_blank" className={output_path === null ? "btn btn-primary disabled":"btn btn-primary"}>Browse</a>
```

Comment:
Would it be smarter to have something like this? (jinja syntax but you get the idea)

```django
<a className="btn btn-primary{% if output_path is None %} disabled{% endif %}"></a>
```